Appendices.md

```diff
@@ -1380,3 +1380,153 @@ is less than $20,000 (excluding interest, if any), the Court will in general, aw
 ## Footnotes
 
 [^17]: Please refer to Order 92, Rule 1 of the Rules of Court (Cap. 322 R 5).
+
+## APPENDIX L
+
+## PRE-ACTION PROTOCOL FOR BUSINESS-TO-BUSINESS DEBT CLAIMS
+
+## Application
+
+> 1.	This Protocol applies to any action that is commenced in the State Courts from 1 October 2019 onwards by a business (the “creditor”) against another business (the “debtor”) arising from a business-to-business claim.
```

Comment:
It's not clear to me why you are indenting the clauses here. you could make it such that it reads..
```
1.	This Protocol applies to any action that is commenced in the State Courts from 1 October 2019 onwards by a business (the “creditor”) against another business (the “debtor”) arising from a business-to-business claim.
```

and then reduce all the indentation by 1 throughout